stages/test/test_kickstart.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+
+import os.path
+import shutil
+import unittest
+from tempfile import mkdtemp
+
+from org_osbuild_kickstart import main as kickstart_main
+
+test_cases = [
+    {
+        "options": {"lang": "en_US.UTF-8"},
+        "expected": "lang en_US.UTF-8",
+    },
+    {
+        "options": {"keyboard": "us"},
+        "expected": "keyboard us",
+    },
+    {
+        "options": {"timezone": "UTC"},
+        "expected": "timezone UTC",
+    },
+    {
+        "options": {
+            "lang": "en_US.UTF-8",
+            "keyboard": "us",
+            "timezone": "UTC",
+        },
+        "expected": "lang en_US.UTF-8\nkeyboard us\ntimezone UTC",
+    },
+]
+
+
+class TestOrgOsbuilderKickstart(unittest.TestCase):
+
+    def setUp(self):
+        self.tmpdir = mkdtemp()
+        self.addCleanup(shutil.rmtree, self.tmpdir)
+
+    def test_kickstart(self):
+        ks_path = "kickstart/kfs.cfg"
+        for tc in test_cases:
```

Comment:
Let's use pytest fixtures for this instead we're already using pytest as the test runner :)

Reply:
Sure thing, I "rewrote" it and went pytest all the way now (hope that was okay)